tsdb/wlog/wlog.go

```diff
@@ -233,6 +234,19 @@ func newWLMetrics(r prometheus.Registerer) *wlMetrics {
 		Name: "writes_failed_total",
 		Help: "Total number of write log writes that failed.",
 	})
+	m.walFileSize = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "storage_size_bytes",
+		Help: "Size of the WAL directory.",
```

Comment:
It can be either WAL or WBL here
```suggestion
		Help: "Size of the write log directory.",
```

Reply:
Done.